### PR TITLE
Add plist to Caddy

### PR DESCRIPTION
**Pull request by miniyarov**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/caddy.rb [hunk 1]
```diff
@@ -27,6 +27,30 @@ def install
            "-o", bin/"caddy", "github.com/mholt/caddy/caddy"
   end
 
+  plist_options :manual => "caddy"
```
Comment (ilovezfs) [inline Formula/caddy.rb]: shouldn't we pass the conf file arg here too?

**Comment by miniyarov** [comment]
@ilovezfs updated with the conf file args

File: Formula/caddy.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def install
            "-o", bin/"caddy", "github.com/mholt/caddy/caddy"
   end
 
-  plist_options :manual => "caddy -conf #{etc}/Caddyfile"
+  plist_options :manual => "caddy -conf $(brew --prefix)/Caddyfile"
```
[the remark this answers is not in the document]
Comment (miniyarov) [reply Formula/caddy.rb]: yes, just noticed that and fixed

**Comment by ilovezfs** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @miniyarov!

**Pull request merged**